examples/messenger_using_kafka/src/kafka_producer.rs

```diff
@@ -0,0 +1,48 @@
+use async_trait::async_trait;
+use log::info;
+use rdkafka::producer::ProducerContext;
+use talos_messenger_actions::kafka::{
+    models::KafkaAction,
+    producer::{KafkaProducer, MessengerProducerDeliveryOpaque},
+};
+use talos_messenger_core::core::{MessengerPublisher, PublishActionType};
+// use talos_messenger::{
+//     core::{MessengerPublisher, PublishActionType},
+//     kafka::producer::{KafkaProducer, MessengerProducerDeliveryOpaque},
+//     models::commit_actions::publish::KafkaAction,
+// };
+
+pub struct MessengerKafkaPublisher<C: ProducerContext + 'static> {
+    pub publisher: KafkaProducer<C>,
+}
+
+#[async_trait]
+impl<C> MessengerPublisher for MessengerKafkaPublisher<C>
+where
+    C: ProducerContext<DeliveryOpaque = Box<MessengerProducerDeliveryOpaque>> + 'static,
+{
+    type Payload = KafkaAction;
+    type AdditionalData = u32;
+    fn get_publish_type(&self) -> PublishActionType {
+        PublishActionType::Kafka
+    }
+
+    async fn send(&self, version: u64, payload: Self::Payload, additional_data: Self::AdditionalData) -> () {
+        info!("[MessengerKafkaPublisher] Publishing message with payload=\n{payload:#?}");
```

Comment:
It's good if we could agree on prefix, so that we use it consistently. Somewhere in the code I used format like this: "MessengerKafkaPublisher.send(): ...". Does not matter really just better when there is one format :)